GDJS/Runtime/runtimeobject.js

```diff
@@ -1357,10 +1357,7 @@ gdjs.RuntimeObject.prototype.separateObjectsWithoutForces = function(objectsList
  * @deprecated
  * @param objectsLists Tables of objects
  */
-gdjs.RuntimeObject.prototype.separateObjectsWithForces = function(objectsLists, len) {
-
-    if ( len == undefined ) len = 10;
-
+gdjs.RuntimeObject.prototype.separateObjectsWithForces = function(objectsLists, dummyVar) {
```

Comment:
Seems that you're right, the value will be erased anyway.
You can remove `dummyVar`, JavaScript is not crashing if extra arguments are passed and not used. (Typescript would error, but not JS)/